ext/sodium/sodium_pwhash.c

```diff
@@ -19,25 +19,14 @@
 #endif
 
 #include "php.h"
-#include "php_libsodium.h"
 #include "ext/standard/php_password.h"
+#include "php_libsodium.h"
+#include "sodium_pwhash_arginfo.h"
 
 #include <sodium.h>
 
 #if SODIUM_LIBRARY_VERSION_MAJOR > 9 || (SODIUM_LIBRARY_VERSION_MAJOR == 9 && SODIUM_LIBRARY_VERSION_MINOR >= 6)
 
-/**
```

Comment:
This block was moved outside of the above `#if`, so replicating this if in the header file is probably needed.